ecodse_funtime_alpha/train.py

```diff
@@ -21,6 +41,28 @@ def batch_dataset(dataset, nepoch, batchsize):
 
 
 def train_loop(dataset, model, optimizer, nepoch, batchsize):
+    """
+    Training loop feeding the mini-batches in the dataset in the model one at a time.
+    Gradient is applied manually on the model using the optimizer.
+
+    Parameters
+    ----------
+    dataset : tf dataset
+        original dataset (unshuffled, not-split into mini-batches)
+    model : tf.keras.Model
+        an initialized model working in eager execution mode
+    optimizer : tf.train.Optimizer
+        tensorflow optimizer (e.g. tf.train.AdamOptimizer()) to train the model
```

Comment:
Code should be in between `` so it looks nice when building the doc. Here it should be`tf.train.AdamOptimizer()`

Reply:
solved. At line 87 as well.